src/compiler/types.ts

```diff
@@ -487,7 +488,7 @@ export const enum SyntaxKind {
     FirstJSDocTagNode = JSDocTag,
     LastJSDocTagNode = JSDocImportTag,
     /** @internal */ FirstContextualKeyword = AbstractKeyword,
-    /** @internal */ LastContextualKeyword = OfKeyword,
+    /** @internal */ LastContextualKeyword = DeferKeyword,

```

Comment:
```suggestion
    /** @internal */ LastContextualKeyword = LastKeyword,

```
Just to prevent needing to edit it again.